data/mods/ssb/moves.ts

```diff
@@ -1335,18 +1397,24 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		accuracy: 100,
 		basePower: 60,
 		category: "Special",
-		desc: "",
-		shortDesc: "",
+		desc: "User gains +1 speed and changes form. Type depends on form",
+		shortDesc: "User gains +1 speed and changes form. Type depends on form",
```

Comment:
```suggestion
		desc: "Has a 100% chance to raise the user's Speed by 1 stage. If the user is a Hoopa in its Confined forme, this move is Psychic type, and Hoopa will change into its Unbound forme. If the user is a Hoopa in its Unbound forme, this move is Dark type, and Hoopa will change into its Confined forme. This move cannot be used successfully unless the user's current form, while considering Transform, is Confined or Unbound Hoopa.",
		shortDesc: "Hoopa: Psychic; Unbound: Dark; 100% +1 Spe. Changes form.",
```
I would also move the speed boost to a secondary effect but meh